src/main/java/com/minecolonies/coremod/util/constants/ToolType.java

```diff
@@ -0,0 +1,56 @@
+package com.minecolonies.coremod.util.constants;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public enum ToolType implements IToolType
+{
+    NONE       ("",        false),
+    PICKAXE    ("pickaxe", true),
+    SHOVEL     ("shovel",  true),
+    AXE        ("axe",     true),
+    HOE        ("hoe",     true),
+    SWORD      ("weapon",  true),
+    BOW        ("bow",     false),
+    FISHINGROD ("rod",     false),
+    SHEARS     ("shears",  false),
+    SHIELD     ("shield",  false);
+
+    static final private Map<String,IToolType> tools = new HashMap<>();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AModifiersOrderCheck)